features-sdk/step_definitions/documents-steps.js

```diff
@@ -131,6 +131,18 @@ Then(/The document "(.*?)" should( not)? exist/, async function (id, not) {
   }
 });
 
+Then('I {string} the following document ids with verb {string}:', async function (action, verb, dataTable) {
```

Comment:
```suggestion
Then(/I "(.*?)" the following document ids( with verb "(.*?)")?:/, async function (action, verb, dataTable) {
```
You can use a regex with cucumber instead of redeclaring an entire new step. I'm not quite sure on how cucumber will pass the arguments to the function, maybe you will have to try 3 to 4 arguments

Reply:
It totally works :o 
Thanks